HW2/index.html

```diff
@@ -1,64 +1,72 @@
-                        nostrum explicabo eveniet mollitia illum deleniti.</p>
-                    <p class="paragraph">Lorem ipsum dolor sit amet consectetur adipisicing elit. Saepe,
-                        ipsam? Maiores incidunt, molestiae quo voluptatum, rem tempora earum
-                        debitis praesentium quis autem dolores ipsam explicabo repudiandae!
-                        Consequatur eum consectetur, ex, quasi eius aperiam qui quis a modi
-                        repellendus accusantium omnis architecto illum nesciunt autem nihil
-                        soluta nobis doloribus sapiente corrupti?</p>
-                </div>
-            </div>
+      <main class="main-content">
+        <div class="left-sidebar">
+          <a href="#" class="left-sidebar__subPart">Ramda</a>
+          <a href="#" class="left-sidebar__subPart">Why Rmada?</a>
+          <a href="#" class="left-sidebar__subPart">What's Different?</a>
+          <a href="#" class="left-sidebar__subPart">Introduction</a>
+          <a href="#" class="left-sidebar__subPart">Philosophy</a>
+        </div>
+        <div class="sub-content">
+          <div class="top-img">
+            <img class="ramda-logo" src="img/image.JPG" alt="Ramda logo" />
+          </div>
+          <div class="content">
+            <p class="paragraph">
+              Lorem, ipsum dolor sit amet consectetur adipisicing elit. Beatae
+              repudiandae neque accusantium nemo assumenda delectus architecto,
+              eveniet iusto iste sed quo ullam nobis repellendus libero vero
+              explicabo rem, quis error. Nisi, maiores illo et expedita,
+              laudantium eaque autem quod delectus vero tenetur nemo accusamus
+              nostrum explicabo eveniet mollitia illum deleniti.
+            </p>
+            <p class="paragraph">
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Saepe,
+              ipsam? Maiores incidunt, molestiae quo voluptatum, rem tempora
+              earum debitis praesentium quis autem dolores ipsam explicabo
+              repudiandae! Consequatur eum consectetur, ex, quasi eius aperiam
+              qui quis a modi repellendus accusantium omnis architecto illum
+              nesciunt autem nihil soluta nobis doloribus sapiente corrupti?
+            </p>
+          </div>
         </div>
-        <div class="back-footer">
-            <div class="back-footer__item">
-                <a href="#" class="back-footer__item--h_blue">Ramda</a>
-                <a href="#" class="back-footer__item--h_blue">Why Rmada?</a>
-                <a href="#" class="back-footer__item--h_blue">What's Different?</a>
-                <a href="#" class="back-footer__item--h_blue">Introduction</a>
-                <a href="#" class="back-footer__item--h_blue">Philosophy</a>
-            </div>
-            <p>Copyright © 2018</p>
+      </main>
+      <footer class="footer">
```

Comment:
Better to use the next example of html, class names, and properties:

https://stackblitz.com/edit/js-gg1cvd?file=index.html,style.css